File: Additional Testing Code/UserInterface/src/app/dynamic-reports/models/report-request.model.ts
export class ReportRequestModel {
       practiceCode: number;
       dateTo: string
       dateFrom: string
       locationCode?: number[];
       month: string;
       dateType: string;
       dataType: string;
       /**
        *
        */
       constructor() {
              this.locationCode = [];
       }
}

File: Additional Testing Code/NPMAPI/Models/ViewModels/ReportRequestModel.cs
namespace NPMAPI.Models.ViewModels
{
    public class ReportRequestModel
    {
        public long PracticeCode { get; set; }
        public string DateTo { get; set; }
        public string DateFrom { get; set; }
        public long[] LocationCode { get; set; }
        public string Date { get; set; }
        public string Month { get; set; }
        public string DateType { get; set; }
        public string DataType { get; set; }
    }
}